src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -274,12 +315,108 @@ export class CalciteDropdown {
   /** trigger elements */
   private triggers: HTMLSlotElement[];
 
+  /** positions menu */
+  private popper: Popper;
+
+  /** menu element */
+  private menuEl: HTMLDivElement;
+
+  /** trigger reference element */
+  private referenceEl: HTMLDivElement;
+
   //--------------------------------------------------------------------------
   //
   //  Private Methods
   //
   //--------------------------------------------------------------------------
 
+  getModifiers(): Partial<StrictModifiers>[] {
+    const flipModifier: Partial<StrictModifiers> = {
+      name: "flip",
```

Comment:
Do we want to allow the user to select "always top" "always bottom" or "auto" (current behavior = "auto" via prop?

Reply:
I think we can leave it as is for now.